server/rest/src/main/java/org/infinispan/rest/logging/Log.java

```diff
@@ -106,4 +106,7 @@
    @LogMessage(level = WARN)
    @Message(value = "The REST invocation [%s] has been deprecated. Please consult the upgrade guide", id = 12026)
    void warnDeprecatedCall(String invocation);
+
+   @Message(value = "Authorization is not enabled on this server", id = 12027)
+   String authorizationNotEnabled();
```

Comment:
@tristantarrant When would you encounter this error message? Is this like if you were connected with the CLI and tried to dynamically modify roles but the Cache Manager doesn't enforce authorization?

Reply:
yes

Reply:
Probably because this is not being sent to the logger, but returned through the REST call

Reply:
I see. Thanks for explaining.